src/dsp_tools/commands/xmlupload/models/lookup_models.py

```diff
@@ -20,12 +19,9 @@ class IntermediaryLookup:
 
 
 @dataclass
-class Lookups:
+class IRILookup:
```

Comment:
The name doesn't really fit the content. The name suggests that the only content of this class is the IRI resolver (=1 thing). But actually, it contains 2 lookups.